libs/utils/analysis/tasks_analysis.py

```diff
@@ -699,15 +734,12 @@ def _plotTaskResidencies(self, axes, tid, signals, is_last=False):
         :param is_last: if True this is the last plot
         :type is_last: bool
         """
-        util_df = self._dfg_trace_event('sched_load_avg_task')
-
-        if 'cluster' in util_df:
-            data = util_df[util_df.pid == tid][['cluster', 'cpu']]
-            for ccolor, clabel in zip('gr', ['LITTLE', 'big']):
-                cdata = data[data.cluster == clabel]
-                if len(cdata) > 0:
-                    cdata.plot(ax=axes, style=[ccolor+'+'], legend=False)
-
+        util_df = self._dfg_task_lt_events()
+        data = util_df[util_df.pid == tid][['cluster', 'cpu']]
```

Comment:
Still missing check for `util_df not None`?
Like the one you have at line 581

Reply:
Ditto